scripts/EN/Harry

```diff
@@ -2424,33 +2424,33 @@ DrawBackground(); // 0x2cde: 00000065
 ShowText((1181)"I begin to leave the room."); // 0x2ce2: 0000100c
 HandleInput(); // 0x2ce8: 00000010
 PlaySoundFx(0x17); // 0x2cec: 00001072
-ShowText((1182)"Jiriririri...Jiriririri..."); // 0x2cf2: 0000100c
+ShowText((1182)"Ri-i-ingg...W3Ri-i-ingg..."); // 0x2cf2: 0000100c
```

Comment:
This is Japanese for the sound of a ringing phone. While "bringg" appears to be most accepted in English, I went with an extended "Ri-i-imgg" to try and convey the sound from the game and not create confusion with the word bring.